src/ansys/mapdl/core/errors.py

```diff
@@ -49,49 +49,73 @@
 )
 
 
-class ANSYSDataTypeError(ValueError):
-    """Raised when and invalid data type is sent to APDLMath"""
+## Abraham class
```

Comment:
```suggestion
## Abstract class
```

Reply:
Who doesn't want a Lincoln in their library? 😄

Reply:
hahahaha... that wasn't a typo.... it just to say this class that all other classes inheritate from ... Abraham from the bible actualy hahahaah... 

I thought I deleted 😅

Reply:
Hahahaha!

Reply:
😂😂